openupgradelib/openupgrade.py

```diff
@@ -2367,3 +2368,88 @@ def set_xml_ids_noupdate_value(env, module, xml_ids, value):
         SET noupdate = %s
         WHERE module = %s AND name in %s
     """, (value, module, tuple(xml_ids),))
+
+def move_to_company_dependent(
+        env,
+        model_name,
+        model_table_name,
+        origin_field_name,
+        destination_field_name,
+    ):
+        """ For each row in a given table, the value of a given field is
+        set in another 'company dependant' field of the same table.
+        Useful in cases when from one version to another one, some field in a
+        model becomes a 'company dependent' field.
+
+        :param model_name: Name of the model.
+        :param model_table_name: Name of the table.
```

Comment:
Put this argument as optional, and if not provided, you get it from `env[model_name]._table`.